plugins/outputs/kinesis/kinesis_test.go

```diff
@@ -546,46 +546,46 @@ func (m *mockKinesisPutRecords) AssertRequests(
 	expected []*kinesis.PutRecordsInput,
 ) {
 
-	require.Equal(t,
+	require.Equalf(t,
```

Comment:
If you want to +1 for your `require`-skill you might want to use `require.Lenf` for length checks. :-) But just a remark not a real request.